paasta_tools/setup_marathon_job.py

```diff
@@ -400,17 +408,43 @@ def log_deploy_error(errormsg, level='event'):
         bounce_health_params
     )
 
+    if new_app_running:
+        protected_draining_tasks = set()
+        if new_app.instances < config['instances']:
+            client.scale_app(app_id=new_app.id, instances=config['instances'], force=True)
+        elif new_app.instances > config['instances']:
+            num_tasks_to_scale = len(new_app.tasks) - config['instances']
```

Comment:
probably easiest to put your `max(..., 0)` here instead of duplicating it 3 times later